File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1049_basico.js
import { question } from "readline-sync";

function main() {
    const INPUT_1 = get_text().toUpperCase();
    const INPUT_2 = get_text().toUpperCase();
    const INPUT_3 = get_text().toUpperCase();

    let output;

    if (INPUT_1 === `VERTEBRADO`) {
        if (INPUT_2 === `AVE`) {
            if (INPUT_3 === `CARNIVORO`) {
                output = `aguia`;
            } else {
                output = `pomba`;
            }
        } else {
            if (INPUT_3 === `ONIVORO`) {
                output = `homem`;
            } else {
                output = `vaca`;
            }
        }
    } else {
        if (INPUT_2 === `INSETO`) {
            if (INPUT_3 === `HEMATOFAGO`) {
                output = `pulga`;
            } else {
                output = `lagarta`;
            }
        } else {
            if (INPUT_3 === `HEMATOFAGO`) {
                output = `sanguessuga`;
            } else {
                output = `minhoca`;
            }
        }
    }
    print(`${output}`);
}

function get_text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1049_basico.cpp
#include <stdio.h>
#include <iostream>
#include <string.h>
using namespace std;

main() {
    string geral, classe, habitos_Alimentares, exemplo;
    cin >> geral;
    cin >> classe;
    cin >> habitos_Alimentares;

    if (geral == "vertebrado") {
        if (classe == "ave") {
            if (habitos_Alimentares == "carnivoro") {
                exemplo = "aguia";
            } else {
                exemplo = "pomba";
            }
        } else {
            if (habitos_Alimentares == "onivoro") {
                exemplo = "homem";
            } else {
                exemplo = "vaca";
            }
        }
    } else {
        if (classe == "inseto") {
            if (habitos_Alimentares == "hematofago") {
                exemplo = "pulga";
            } else {
                exemplo = "lagarta";
            }
        } else {
            if (habitos_Alimentares == "hematofago") {
                exemplo = "sanguessuga";
            } else {
                exemplo = "minhoca";
            }
        }
    }
    cout << exemplo.c_str() << endl;
}